move-maker.js

```diff
@@ -16,8 +16,31 @@
         ];
 */
 function validateMove(move, board) {
-    // Implement this at the end if you have time, otherwise you can help your teammates!
+  // Implement this at the end if you have time, otherwise you can help your teammates!
+  const getSplitedArray = move.split(',');
+
+  const moveIncludesComma = move.includes(',');
+  const checkIfArrayLengthTwo = getSplitedArray.length === 2;
+
+  const checkFirstValue = getSplitedArray[0] >= 1 && getSplitedArray[0] <= 3;
+  const checkSecondValue = getSplitedArray[1] >= 1 && getSplitedArray[1] <= 3;
+  const ifMoveValid = checkFirstValue && checkSecondValue;
+
+  const isUnderLineOnBoard =
+    board[Math.max(getSplitedArray[0] - 1, 0)][
```

Comment:
This line will cause errors if you type in invalid move. For example, `test` (move without comma), `1.1,2.1` (move which are not integers)

Reply:
I referred to the Barath video and rewrote the coding. So `test` and `1.1,2.1` are invalid now.
Do you have any suggestion about the twice `return false`? Because we should avoid the "repeat coding", right?
But I inserted the 2nd "if statement" for sure `move[0]` and `move[2]` both are '1'||'2'||'3'... otherwise it will pop up error msg